geoips/geoips_utils.py

```diff
@@ -73,19 +74,20 @@ def load_all_yaml_plugins():
     in ``.yaml``. Read each plugin file
     """
     # Load all entry points for plugin packages
+    import pickle  # nosec
 
     plugin_packages = get_entry_point_group("geoips.plugin_packages")
     yaml_plugins = {}
     for pkg in plugin_packages:
-        pkg_plug_path = str(resources.files(pkg.value) / "registered_plugins.yaml")
+        pkg_plug_path = str(resources.files(pkg.value) / "registered_plugins")
         if not os.path.exists(pkg_plug_path):
             raise PluginRegistryError(
                 f"Plugin registry {pkg_plug_path} did not exist, "
                 "please run 'create_plugin_registries'"
             )
         # This will include all plugins, including schemas, yaml_based,
         # and module_based plugins.
-        registered_plugins = yaml.safe_load(open(pkg_plug_path, "r"))
+        registered_plugins = pickle.load(open(pkg_plug_path, "rb"))  # nosec
         # Only pull the "yaml_based" plugins here.
```

Comment:
Should this support both YAML and pickle?

Reply:
I would say no. This is done for efficiency's purpose, and loading yaml takes much longer. Since this is on the backend of GeoIPS, I'd definitely support going the quicker route rather than the longer route.

Reply:
Replaced with json.  Resolving.